### PR TITLE
Bugfixes and addition after OSCAP run

### PR DESCRIPTION
**Pull request by bbaassssiiee**

### CODE REVIEW

File: tasks/cat2.yml [hunk 1]
```diff
@@ -245,6 +245,31 @@
       - patch
       - auditd
 
+- name: "MEDIUM | V-54381 | AUDIT | The audit system must switch the system to single-user mode when available audit storage volume becomes dangerously low."
```
Comment (defionscode) [inline tasks/cat2.yml]: This is out of order.

File: tasks/cat2.yml [hunk 2]
```diff
@@ -481,6 +506,35 @@
       - V-38484
       - sshd
 
+- name: "MEDIUM | V-51875 | AUDIT | The operating system, upon successful logon, must display to the user the date and time of the last logon or access via ssh."
+  shell: "grep 'pam_lastlog.so' /etc/pam.d/system-auth"
+  ignore_errors: yes
```
Comment (defionscode) [inline tasks/cat2.yml]: change to `failed_when: no` instead of ignore errors

File: tasks/cat2.yml [hunk 3]
```diff
@@ -481,6 +506,35 @@
...
+  changed_when: no
+  always_run: yes
+  tags:
+      - cat2
+      - medium
+      - V-51875
+      - audit
+      - sshd
+
+- name: "MEDIUM | V-51875 | PATCH | The operating system, upon successful logon, must display to the user the date and time of the last logon or access via ssh."
```
Comment (defionscode) [inline tasks/cat2.yml]: also out of order

File: tasks/cat2.yml [hunk 4]
```diff
@@ -1768,6 +1822,7 @@
   shell: "chkconfig --list | sed -n '/xinetd based services/,$p'"
   changed_when: no
   always_run: yes
+  ignore_errors: yes
```
Comment (defionscode) [inline tasks/cat2.yml]: change to failed_when

File: tasks/cat2.yml [hunk 5]
```diff
@@ -2029,6 +2084,7 @@
   shell: "sysctl kernel.randomize_va_space | awk '{print $3}'"
   changed_when: no
   always_run: yes
+  ignore_errors: yes
```
Comment (defionscode) [inline tasks/cat2.yml]: change to failed_when

File: tasks/cat2.yml [hunk 6]
```diff
@@ -3052,6 +3108,118 @@
   ignore_errors: yes
   tags: [ 'cat2' , 'V-38691' , 'bluetooth' ]
 
+- name: V-38619 Medium  There must be no .netrc files on the system
```
Comment (defionscode) [inline tasks/cat2.yml]: These do not follow the style guide in https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md

**Comment by defionscode** [comment]
Also, there is quite a bit of merge conflicts, you need to rebase as I see you're contributing things which have already been addressed in the Master branch

**Comment by bbaassssiiee** [comment]
Ah, at first there was only a devel branch. Do we rebase off master now?

**Comment by defionscode** [comment]
Oops sorry, I meant devel

**Comment by mliudev** [comment]
The sshd fixes are breaking so +1.

**Comment by bbaassssiiee** [comment]
Another day wasted.

**Pull request closed without merging**